aqimon/config.py

```diff
@@ -52,8 +54,11 @@ def get_config_from_env() -> Config:
         retention_minutes=int(os.environ.get("AQIMON_RETENTION_MINUTES", DEFAULT_CONFIG.retention_minutes)),
         reader_type=os.environ.get("AQIMON_READER_TYPE", DEFAULT_CONFIG.reader_type),
         usb_path=os.environ.get("AQIMON_USB_PATH", DEFAULT_CONFIG.usb_path),
-        usb_sleep_time_sec=int(os.environ.get("AQIMON_USB_SLEEP_TIME_SEC", DEFAULT_CONFIG.usb_sleep_time_sec)),
+        sleep_sec_between_reads=int(
+            os.environ.get("AQIMON_SLEEP_SEC_BETWEEN_READS", DEFAULT_CONFIG.sleep_sec_between_reads)
```

Comment:
Note that this is renamed, so its a breaking change.  Possible you set this to do local dev.